dali/core/mm/cuda_vm_resource_test.cc

```diff
@@ -187,6 +187,7 @@ class VMResourceTest : public ::testing::Test {
     void *p1 = res.allocate(block_size);  // allocate one block
     void *p2 = res.allocate(block_size);  // allocate another block
     res.deallocate(p2, block_size);       // now free the second block
+    res.flush_deferred();
```

Comment:
This test uses the one with deferred deallocation - that's why this line is needed - otherwise we might not see the result of the deallocation and the test expectations will fail.